src/oemof/solph/_energy_system.py

```diff
@@ -20,6 +21,37 @@
 import numpy as np
 import pandas as pd
 from oemof.network import energy_system as es
+from oemof.tools import debugging
+
+
+def _add_periods(periods):
```

Comment:
Yes, you are right. It is a check, but used to be something different. I forgot to adjust the naming. Nevermind, since your improvements from #935 make it obsolete anyways.